config/load_test.go

```diff
@@ -1,9 +1,10 @@
 package config_test
 
 import (
-	"go.arcalot.io/log/v2"
 	"testing"
 
+	"go.arcalot.io/log/v2"
+
 	"go.arcalot.io/lang"
 	"go.flow.arcalot.io/engine/config"
```

Comment:
Dustin, did you make this change, or did `go fmt` do this for you?

I agree that the `log` reference was incorrectly positioned in the old code, but it doesn't look quite correctly positioned in the new code, either:  there shouldn't be a blank line separating `log` from the rest of the Arcalot dependencies, and I think that `log` should follow `lang`.

`engine_test.go` is suffering similarly but even more acutely.

I suggest running `go fmt` again...if that doesn't fix it, then nevermind....

Reply:
I think vscode actually auto-placed it. Running `go fmt` does not result in any changes for me.

Reply:
😛